src/components/Button/Button.plugin.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const _ = require('lodash')
+const chroma = require('chroma-js')
+const defaultConfig = require('tailwindcss/defaultConfig')()
+
+function defaultOptions() {
+  return {
+    borderRadius: '1.5rem',
+    fontWeight: '600',
+    lineHeight: '1.25',
+    fontSize: '1rem',
+    padding: '.5rem 1rem',
+    colors: {
+      white: {
+        background: defaultConfig.colors.white,
```

Comment:
use `defaultConfig.backgroundColors.white` this way if the backhground colors diverge from the brand colors for any reason it will update properly

Reply:
I don't think `backgroundColors` exists in the default config.